relays/client-millau/src/lib.rs

```diff
@@ -44,11 +44,10 @@ impl Chain for Millau {
 	type Index = millau_runtime::Index;
 	type SignedBlock = millau_runtime::SignedBlock;
 	type Call = millau_runtime::Call;
+	type Balance = millau_runtime::Balance;
 }
 
 impl ChainWithBalances for Millau {
```

Comment:
Can we eradicate this trait completely now and move the `account_info_storage_key` function to `Chain` as well?

Reply:
This trait 'protects' us from chains that aren't storing `pallet_balances::AccountData` directly in the storage. I could imagine that there may be additional info stored for every account - account storage root, ref count, ... And `AccountData` is either stored as a part of this data, or is ephemeral. By 'protect' here I mean that we're able to work with such chains, but some functionality (right now it is only relay guards) is unavailable. Moving `account_info_storage_key` would mean that we are not supporting such chains anymore. At least we can't 'detect' supported chains at compilation time.

That said, if you feel that this trait may be removed - please do that in a separate PR. I feel that the current PR itself introduces a quite important change, which deserves its own discussion.